torch/serialization.py

```diff
@@ -103,6 +103,29 @@ def storage_to_tensor_type(storage):
 def save(obj, f, pickle_module=pickle, pickle_protocol=DEFAULT_PROTOCOL):
     """Saves an object to a disk file.
 
+    There are two main approaches for serializing and restoring a model.
```

Comment:
No! People that use the functional style or serialized other stuff should use it to! It has nothing to do with mm.